features/step_definitions/outcome.rb

```diff
@@ -50,3 +50,7 @@
 Alors("l'image intégrée en source {string} est légendée par son titre : {string}") do |image, image_caption|
   expect(find('.sources')).to have_content image_caption
 end
+
+Alors('le document comporte la vidéo avec pour titre {string}') do |title|
+  expect(page).to have_content title
```

Comment:
Did you really get this test to pass?
It seems that iframes are not loaded by Capybara. Hence you have to put iframes-related actions inside `within_frame do <action> end`.